### PR TITLE
Fix sqlite column update

### LINKED ISSUE #2794: Error: SQLITE_ERROR: near "ALTER": syntax error
**Informations**
- **Node.js version**: v10.14.2
- **NPM version**:  6.4.1
- **Strapi version**: 3.0.0-alpha.23.1
- **Database**: SQLite
- **Operating system**: MacOS
- **(Optional) Link to your Project**: -

**What is the current behavior?**

When I update a content type, I have `Error: SQLITE_ERROR: near "ALTER": syntax error` error

**Steps to reproduce the problem**

1. Create a content type `article`
2. Add `title` attribute (string)
3. Save the content type
4. Visit an other page to ovoid this issue https://github.com/strapi/strapi/issues/2793
5. Add `content` attribute (text)
6. Save the content type

**What is the expected behavior?**

I should not have an error.

**Suggested solutions**

Source of the issue https://github.com/strapi/strapi/blob/master/packages/strapi-hook-bookshelf/lib/index.js#L625

### PR DESCRIPTION
**Pull request by lauriejim**
**My PR is a:**
- [ ] 💥 Breaking change
- [x] 🐛 Bug fix #2794
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

**Main update on the:**
- [ ] Admin
- [ ] Documentation
- [x] Framework
- [ ] Plugin

**Manual testing done on the following databases:**
- [ ] Not applicable
- [ ] MongoDB
- [x] MySQL
- [x] Postgres
- [x] SQLite

**Description:**

When you update / create a new attribute on SQLite the function we use (`ALTER COLUMN`) doesn't work on SQLite but works with MySQL and Postgres. 

So we manage attribute creation / update on SQLite.

### CODE REVIEW

File: packages/strapi-hook-bookshelf/lib/index.js [hunk 1]
```diff
@@ -638,6 +644,28 @@ module.exports = function(strapi) {
...
                       ));
 
+                      if (sqlite3Change) {
+                        await createTable(`tmp_${table}`);
+
+                        try {
+                          await ORM.knex.raw(`INSERT INTO ${quote}tmp_${table}${quote}(${Object.keys(attributes).join(' ,')}) SELECT ${Object.keys(attributes).join(' ,')} FROM ${quote}${table}${quote}`);
+                        } catch (err) {
+                          console.log('Warning!');
+                          console.log('We can\'t migrate your data caused by the following error');
```
Comment (derrickmehaffy) [inline packages/strapi-hook-bookshelf/lib/index.js]: "We can't migrate your data due to the following error"

File: packages/strapi-hook-bookshelf/lib/index.js [hunk 2]
```diff
@@ -638,6 +644,28 @@ module.exports = function(strapi) {
...
+                          console.log();
+                          console.log(err);
+                          console.log();
+                          console.log(`We created a new table "tmp_${table}" with your last update.`);
```
Comment (derrickmehaffy) [inline packages/strapi-hook-bookshelf/lib/index.js]: "We created a new table tmp_${table} with your latest changes."

File: packages/strapi-hook-bookshelf/lib/index.js [hunk 3]
```diff
@@ -638,6 +644,28 @@ module.exports = function(strapi) {
...
+                          console.log(`We suggest you manually migrate your data from "${table}" to "tmp_${table}" and then to DROP and RENAME tables.`);
```
Comment (derrickmehaffy) [inline packages/strapi-hook-bookshelf/lib/index.js]: "... DROP and RENAME the tables."

**Comment by lauriejim** [comment]
I'm fixing @derrickmehaffy feedback then I merge this PR

**Pull request merged**